Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent { label 'executor-v2' }
+
+  options {
+    timestamps()
+    buildDiscarder(logRotator(numToKeepStr: '30'))
+  }
+
```

Comment:
Can we add the changelog parser here too? The [docker usage](https://github.com/cyberark/parse-a-changelog/#usage) is quite nice now